roles/generate-jenkins/templates/DOCUMENTATION.j2

```diff
@@ -3,32 +3,14 @@
 ---
 title: {{ project_name }}
 ---
-<!-- DO NOT EDIT THIS FILE MANUALLY  -->
-<!-- Please read the {{ project_github_asset }}/.github/CONTRIBUTING.md -->
-
+{% include "README_SNIPPETS/DO_NOT_EDIT.j2" | trim %}
 {% if project_deprecation_status %}
-# DEPRECATION NOTICE
-
-This image is deprecated. We will not offer support for this image and it will not be updated.
-{% if project_deprecation_message is defined %}{{ project_deprecation_message }}{% endif %}
+{% include "README_SNIPPETS/DEPRECATION.j2" | trim %}
```

Comment:
This should be a admonition on docs

Reply:
Fixed in https://github.com/linuxserver/docker-jenkins-builder/pull/217/commits/22f544cc406c43c6641ccdc4c32fa84a16ad7f30